priam/src/main/java/com/netflix/priam/backup/AbstractBackup.java

```diff
@@ -201,40 +200,4 @@ public boolean isValidBackupDir(File keyspaceDir, File columnFamilyDir, File bac
      * Adds Remote path to the list of Remote Paths
      */
     protected abstract void addToRemotePath(String remotePath);
-
-    @Override
```

Comment:
We were not using this as BackupEvent is done by lower layer (S3BaseFileSystem) for SNS notifications. Cleanup of dead code.